scripts/minikube.sh

```diff
@@ -162,6 +162,7 @@ CONTAINER_CMD=${CONTAINER_CMD:-"docker"}
 MEMORY=${MEMORY:-"4096"}
 CPUS=${CPUS:-"$(nproc)"}
 VM_DRIVER=${VM_DRIVER:-"virtualbox"}
+CNI=${CNI:-"bridge"}
```

Comment:
@nixpanic whats the default CNI ? if its not `bridge` may be the change for CNI to `bridge` causing the E2E to take lots of time to complete , iow, we are hitting some performance issues being the CNI bridge

Reply:
The default CNI is "auto"... There seems to be some logic in minikube somewhere that decides what CNI to use (maybe dependent on the Kubernetes version and hypervisor?).

It is possible that the minikube VM needs more resources with this rebase. Those are settings in the `ci/centos` branch, so we could increase those, depending on the current values and bare metal systems in the CI.